src/cli/commands/commands.js

```diff
@@ -1,10 +1,22 @@
 'use strict'
 
 const Command = require('ronin').Command
+const path = require('path')
+const ronin = require('ronin')
 
 module.exports = Command.extend({
   desc: '',
 
   run: name => {
+    const cli = ronin(path.resolve(__dirname, '..'))
```

Comment:
True, but couldn't find out any other way of doing it :worried: 
Requiring `src/cli/index` didn't work as it only got the commands afterwards. 

However this is not a full ronin as `.run()` isn't called :)